CONTRIBUTING.md

```diff
@@ -0,0 +1,109 @@
+# How to Contribute to Jaeger
```

Comment:
please add a section to README referencing this file

Reply:
Yes, Sure!